argzoo/cifar100/cifar100_fedcac_resnet_config.py

```diff
@@ -0,0 +1,65 @@
+from easydict import EasyDict
+import argparse
+def args_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--tau', type=float, default=0.4, help='tau')
+    parser.add_argument('--beta', type=int, default=170, help='beta')
+    parser.add_argument('--noniid', type=str, default='dirichlet', help='noniid type')
+    parser.add_argument('--alpha', type=float, default=0.5, help='alpha')
+    parser.add_argument('--seed', type=int, default=1, help='seed')
+    args = parser.parse_args()
+    args.alpha = args.alpha if args.noniid == 'dirichlet' else int(args.alpha)
+    return args
+
+args = args_parser()
+tau = args.tau
+beta = args.beta
+noniid = args.noniid
+alpha = args.alpha
+seed = args.seed
+
+exp_args = dict(
+    data=dict(
+        dataset='cifar100',
+        data_path='./data/CIFAR100',
+        sample_method=dict(name=noniid, alpha=alpha, train_num=500, test_num=100),
+        transforms=dict(
+            horizontal_flip=dict(p=0.5),
```

Comment:
check the data-augmentation carefully, maybe this can be removed or simpified by applying default data-aug.